src/panels/config/integrations/integration-panels/zwave_js/zwave_js-config-dashboard.ts

```diff
@@ -211,22 +235,175 @@ class ZWaveJSConfigDashboard extends LitElement {
                 </ha-card>
                 <ha-card header="Diagnostics">
                   <div class="card-content">
-                    ${this.hass.localize(
-                      "ui.panel.config.zwave_js.dashboard.driver_version"
-                    )}:
-                    ${this._network.client.driver_version}<br />
-                    ${this.hass.localize(
-                      "ui.panel.config.zwave_js.dashboard.server_version"
-                    )}:
-                    ${this._network.client.server_version}<br />
-                    ${this.hass.localize(
-                      "ui.panel.config.zwave_js.dashboard.home_id"
-                    )}:
-                    ${this._network.controller.home_id}<br />
-                    ${this.hass.localize(
-                      "ui.panel.config.zwave_js.dashboard.server_url"
-                    )}:
-                    ${this._network.client.ws_server_url}<br />
+                    <div class="row">
```

Comment:
I feel like `<mwc-list-item twoline hasmeta>` might be nicer here. Something like this:

![image](https://user-images.githubusercontent.com/1444314/169448372-0ddbacf8-b0a6-45f2-adb5-79743ee9966a.png)

But instead of `>` it would have the value.

Reply:
thanks for the suggestion, this looks a lot better: 
<img width="570" alt="image" src="https://user-images.githubusercontent.com/7243222/169623939-ed8f1633-d888-420e-a34f-6ffbe5cf06d7.png">

Only downside is that it spaces things out a lot so you can't get as much on the screen

Reply:
I dropped the `height` to 60 px and all is good